src/xcfun.cpp

```diff
@@ -729,3 +729,121 @@ int xc_is_gga(xc_functional fun) { return (fun->depends & XC_GRADIENT); }
 int xc_is_metagga(xc_functional fun) {
   return (fun->depends & (XC_LAPLACIAN | XC_KINETIC));
 }
+
+/*! brief host program-friendly setup of the functional
```

Comment:
The Doxygen directives start with a `\` or an `@` like `\brief`. Also I think you return the functional so it's `\param[in,out]` rather than `\param[in]`.